UnityProject/Assets/Scripts/PlayGroups/Input/InputController.cs

```diff
@@ -54,6 +54,7 @@ private void Update()
 			CheckHandSwitch();
 			CheckClick();
 			CheckAltClick();
+			CheckSuicide();

```

Comment:
Can you make sure it gets disabled when "release" in gamedata is checked in the inspector?